### ISSUE TITLE
I don't find these here on my Windows 10 1909 with both Windows PowerShell 5.1 and PowerShell 7.0 Installed

### ISSUE THREAD

**Issue by jsilverm** [MicrosoftDocs/PowerShell-Docs#5965]
I was following along with 3rd edition Jones and Hicks, chapter 10, and couldn't find the format.ps1xml files where they suggested they would be, so I came to this page to find where they'd moved to in 5.1 and 7.0, and it says they should be there.  Unless they are hidden -- nope, checked that -- I can find various ones under /modules/ subdirectories, such as Diagnostics.format.ps1xml and PSReadLine.format.ps1xml, which might or might not work just as well for these example purposes, I don't know.

Maybe they are better hidden away because boneheads keep editing them by mistake, but this page is useless for me on Windows 10 with 5.1 and 7.0 PowerShell installed normally.  Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f71b7da0-8de9-3c01-bf6d-d8263eab42d2
* Version Independent ID: f914b9ac-d8f5-545b-cd54-0d757774aacd
* Content: [Windows PowerShell Formatting Files - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/developer/format/powershell-formatting-files?view=powershell-7)
* Content Source: [reference/docs-conceptual/developer/format/powershell-formatting-files.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/developer/format/powershell-formatting-files.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

**Comment by jsilverm** [comment]
I see (only now) that this is considered legacy.  I am not sure what the new functionality is, and if all of this stuff is truly deprecated in modern PowerShell versions, or there are just backlogs in documenting features.  I will move on, but will note that there are still many blogs/webpages/tutorials on the web that will have people looking for these files in $PSHome, as well as the excellent and popular book referred to above.  Thanks.

**Comment by doctordns** [comment]
In PowerShell 7, object formattiong is no longer done with XML files as was the case in Windows  PowerShell. You can create your own to format your own objects or to over ride the default formatting of any object.

**Comment by sdwheeler** [comment]
See https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_format.ps1xml?view=powershell-7#creating-new-formatps1xml-files

To create a Format.ps1xml file to define a custom view, use the Get-FormatData and Export-FormatData cmdlets. Use a text editor to edit the file. The file can be saved to any directory that PowerShell can access, such as a subdirectory of $HOME.

**Comment by jsilverm** [comment]
I found this quote "Beginning in PowerShell 6, the default views are defined in PowerShell C# source code. The *.format.ps1xml files from PowerShell 5.1 and earlier versions don't exist in PowerShell 6 and later versions." on page:
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/format-table?view=powershell-7
That makes it very clear.  It is too easy to land first on this page or one of the adjacent pages due to large amounts of printed and web stuff referring to the old way, I think repeating the above small quote on the 6.2+ or 7.0+ version of these pages would reduce confusion for people coming in from the cold as I did today...It is true that 3 levels above this page in the organization one sees "Legacy PowerShell SDK" in the tree on the left side, but landing on these pages and reading them top down, it is just too easy to think they still  apply.  Cheers!
